.prettierignore

```diff
@@ -1,11 +1,11 @@
-node_modules
-**/node_modules
-
-dist
-coverage
-
-openshift
-sonar-runner
-uploads
-
-.vscode
+node_modules
+**/node_modules
+
+dist
+coverage
+
+openshift
+sonar-runner
+uploads
+
+.vscode

```

Comment:
Looks like this file got moved from `/api` to `/`, I assume accidentally.  Just needs to get moved back